### PR TITLE
[WIP] Add Accept All Changes to External Changes Dialog 

### PR DESCRIPTION
**Pull request by danielabajirov**
Fix for issue https://github.com/koppor/jabref/issues/280

Added a new ChoiceBox "Accept all changes" to the JabRefChangeDisplay that allows to select all the change and modify the status of the ChoiceBox from false to true so that all the cb now are selected when the computation is completed.

----

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Screenshots added (for bigger UI changes)
- [ ] Manually tested changed features in running JabRef
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

File: src/main/java/org/jabref/gui/collab/ChangeDisplayDialog.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
     private final JTree tree;
     private final JPanel infoPanel = new JPanel();
     private final JCheckBox cb = new JCheckBox(Localization.lang("Accept change"));
+    private final JCheckBox cb2 = new JCheckBox(Localization.lang("Accept all change"));
```
Comment (stefan-kolb) [inline src/main/java/org/jabref/gui/collab/ChangeDisplayDialog.java]: Better: "Accept all changes"?!

Comment (danielabajirov) [reply src/main/java/org/jabref/gui/collab/ChangeDisplayDialog.java]: Oh right ahah my bad.

**Comment by Siedlerchr** [comment]
Changelog? And it looks like this is the same PR as  #3731

**Comment by danielabajirov** [comment]
I didnt read it so the #3731 but yes the idea is the same now that i gave a look.

**Comment by koppor** [comment]
Could you add a screenshot so that we can decide which one looks better? Codewise, this is beter than #3731.

**Comment by danielabajirov** [comment]
Yep but after i fix a couple of things because i read the #3731 and need to implement the "deselect all changes" ChoiceBox and maybe i had an other idea. I'll post it as soon as i finish.

**Pull request closed without merging**